src/display.rs

```diff
@@ -236,7 +236,6 @@ impl Display {
     /// Currently this only works on X11 with XComposite and XDamage extensions available.
     ///
     /// Deprecated since GDK 3.16
-    #[cfg(not(feature = "gdk_3_16"))]
```

Comment:
How do you handle this then ? I didn't see anything who did remove this function from the build if `gdk` version was >= 3.16.